From ed05f99e637d241950f1e22b4e16a667e7f40f02 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 28 Dec 2024 05:29:41 -0800
Subject: [PATCH] Explicitly set version variable

---
 platforms/libretro/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platforms/libretro/Makefile b/platforms/libretro/Makefile
index 7d0bdf2f..c5db55c9 100644
--- a/platforms/libretro/Makefile
+++ b/platforms/libretro/Makefile
@@ -333,7 +333,7 @@ endif
 
 GIT_VERSION ?= $(shell git describe --abbrev=7 --dirty --always --tags || echo unknown)
 ifneq ($(GIT_VERSION), unknown)
-	CXXFLAGS += -DEMULATOR_BUILD=\"$(GIT_VERSION)\"
+	CXXFLAGS += -DEMULATOR_BUILD=\"3.8.5\"
 endif
 
 include Makefile.common
-- 
2.43.0

