From 79a21c4435d57d76c13e7a6c4ecd7505a52ebd78 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 14 Dec 2024 23:29:08 -0800
Subject: [PATCH] Hard-code version number

---
 platforms/libretro/libretro.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platforms/libretro/libretro.cpp b/platforms/libretro/libretro.cpp
index a01ca8a..be58df2 100644
--- a/platforms/libretro/libretro.cpp
+++ b/platforms/libretro/libretro.cpp
@@ -137,7 +137,7 @@ void retro_get_system_info(struct retro_system_info *info)
 {
     memset(info, 0, sizeof(*info));
     info->library_name     = GEARBOY_TITLE;
-    info->library_version  = GEARBOY_VERSION;
+    info->library_version  = "3.5.0";
     info->need_fullpath    = false;
     info->valid_extensions = "gb|dmg|gbc|cgb|sgb";
 }
-- 
2.43.0

